### PR TITLE
[Finish] Improving German localization

### PR DESCRIPTION
**Pull request by enterih**
I found some stuff to improve the translation. I'm not done yet but i think it's ready to get an eye on it. 

Hope it will be used :)

Greets Sprungente/enterih

### CODE REVIEW

File: Essentials/src/messages_de.properties [hunk 1]
```diff
@@ -127,18 +127,18 @@ extinguishOthers=\u00a76Du hast {0}\u00a76 gel\u00f6scht.
 failedToCloseConfig=Fehler beim Schlie\u00dfen der Konfiguration {0}.
 failedToCreateConfig=Fehler beim Erstellen der Konfiguration {0}.
 failedToWriteConfig=Fehler beim Schreiben der Konfiguration {0}.
-false=\u00a74falsch\u00a7r
+false=\u00a74Nein\u00a7r
```
Comment (mdcfe) [inline Essentials/src/messages_de.properties:130]: Is this actually more appropriate than `falsch`?

**Comment by enterih** [comment]
Yes i think
True is translated with „ja“. I recommend also change true and false into yes and no

In my opinion it sounds in the most situation like /whois better.

Please don‘t merge it. It isn‘t finished yet

Am 08.10.2018 um 10:46 schrieb md678685 <notifications@github.com<mailto:notifications@github.com>>:


@md678685 commented on this pull request.

________________________________

In Essentials/src/messages_de.properties<https://github.com/EssentialsX/Essentials/pull/2198#discussion_r223288208>:

> @@ -127,18 +127,18 @@ extinguishOthers=\u00a76Du hast {0}\u00a76 gel\u00f6scht.
 failedToCloseConfig=Fehler beim Schlie\u00dfen der Konfiguration {0}.
 failedToCreateConfig=Fehler beim Erstellen der Konfiguration {0}.
 failedToWriteConfig=Fehler beim Schreiben der Konfiguration {0}.
-false=\u00a74falsch\u00a7r
+false=\u00a74Nein\u00a7r


Is this actually more appropriate than falsch?

File: Essentials/src/messages_de.properties (hunk 1 above)
Comment (enterih) [reply Essentials/src/messages_de.properties:130]: in situation like /whois 
yes a lot :)

**Review by mdcfe — DISMISSED** [review]
I'm happy to merge this when you're ready.

**Comment by enterih** [comment]
@md678685 it's ready to merge
I wanna maintanance in the future if it's ok for you.

**Review by mdcfe — DISMISSED** [review]

**Pull request merged**